scripts/eurostat/health_determinants/fruits_vegetables/download.py

```diff
@@ -0,0 +1,98 @@
+
+
+def download_files(download_directory: str) -> None:
+    """
+    This Method calls the download function from the commons directory
+    to download all the input files.
+
+    Args:
+        download_directory (str):Location where the files need to be downloaded.
+
+    Returns:
+        None
+    """
+    # List to provide the URLs of input files to download script.
+    INPUT_URLS = ["https://ec.europa.eu/eurostat/estat-navtree-portlet-prod/"+\
+       "BulkDownloadListing?file=data/hlth_ehis_fv3e.tsv.gz",
+       "https://ec.europa.eu/eurostat/estat-navtree-portlet-prod/"+\
+           "BulkDownloadListing?file=data/hlth_ehis_fv3i.tsv.gz",
+       "https://ec.europa.eu/eurostat/estat-navtree-portlet-prod/"+\
+           "BulkDownloadListing?file=data/hlth_ehis_fv3u.tsv.gz",
+       "https://ec.europa.eu/eurostat/estat-navtree-portlet-prod/"+\
+           "BulkDownloadListing?file=data/hlth_ehis_fv1e.tsv.gz",
+       "https://ec.europa.eu/eurostat/estat-navtree-portlet-prod/"+\
+           "BulkDownloadListing?file=data/hlth_ehis_fv1u.tsv.gz",
+       "https://ec.europa.eu/eurostat/estat-navtree-portlet-prod/"+\
+           "BulkDownloadListing?file=data/hlth_ehis_fv3b.tsv.gz",
+       "https://ec.europa.eu/eurostat/estat-navtree-portlet-prod/"+\
+           "BulkDownloadListing?file=data/hlth_ehis_fv3c.tsv.gz",
+       "https://ec.europa.eu/eurostat/estat-navtree-portlet-prod/"+\
+           "BulkDownloadListing?file=data/hlth_ehis_fv3d.tsv.gz",
+       "https://ec.europa.eu/eurostat/estat-navtree-portlet-prod/"+\
+           "BulkDownloadListing?file=data/hlth_ehis_fv3m.tsv.gz",
+       "https://ec.europa.eu/eurostat/estat-navtree-portlet-prod/"+\
+           "BulkDownloadListing?file=data/hlth_ehis_fv1b.tsv.gz",
+       "https://ec.europa.eu/eurostat/estat-navtree-portlet-prod/"+\
+           "BulkDownloadListing?file=data/hlth_ehis_fv1c.tsv.gz",
+       "https://ec.europa.eu/eurostat/estat-navtree-portlet-prod/"+\
+           "BulkDownloadListing?file=data/hlth_ehis_fv1i.tsv.gz",
+       "https://ec.europa.eu/eurostat/estat-navtree-portlet-prod/"+\
+           "BulkDownloadListing?file=data/hlth_ehis_fv1m.tsv.gz",
+       "https://ec.europa.eu/eurostat/estat-navtree-portlet-prod/"+\
+           "BulkDownloadListing?file=data/hlth_ehis_fv1d.tsv.gz",
+       "https://ec.europa.eu/eurostat/estat-navtree-portlet-prod/"+\
+           "BulkDownloadListing?file=data/hlth_ehis_fv7e.tsv.gz",
+       "https://ec.europa.eu/eurostat/estat-navtree-portlet-prod/"+\
+           "BulkDownloadListing?file=data/hlth_ehis_fv7m.tsv.gz",
+       "https://ec.europa.eu/eurostat/estat-navtree-portlet-prod/"+\
+           "BulkDownloadListing?file=data/hlth_ehis_fv7i.tsv.gz",
+       "https://ec.europa.eu/eurostat/estat-navtree-portlet-prod/"+\
+           "BulkDownloadListing?file=data/hlth_ehis_de7.tsv.gz",
+       "https://ec.europa.eu/eurostat/estat-navtree-portlet-prod/"+\
+           "BulkDownloadListing?file=data/hlth_ehis_de8.tsv.gz"
+       ]
```

Comment:
INPUT_URLS = ['https://ec.europa...' + file for  file in file_names]
where filen_names will have list of  only file names in it.